monasca-python/build.sh

```diff
@@ -1,122 +0,0 @@
-#!/bin/sh
```

Comment:
LOL, :1st_place_medal: for me.
I ran a ```find``` command and just removed everything that popped out...ah :D